spec/controllers/items_controller_spec.rb

```diff
@@ -80,41 +72,37 @@
           expect { post_create(name: "name", wishlist_id: wishlist.id) }.to change(Item, :count).by(1)
         end
 
-        include_examples "redirects to the wishlist page"
+        include_examples "redirects to", :wishlist
 
-        it "shows a notice" do
-          expect(flash[:notice]).not_to be_nil
-        end
+        include_examples "shows a flash message", :notice
       end
 
       context "with invalid parameters" do
-        before { post_create(name: nil, wishlist_id: wishlist.id) }
+        subject! { post_create(name: nil, wishlist_id: wishlist.id) }
```

Comment:
"It's instantiated lazily. That is, the implicit instantiation of the described class or the execution of the block passed to subject doesn't happen until subject or the named subject is referred to in an example. If you want your explict subject to be instantiated eagerly (before an example in its group runs), say subject! instead of subject."